src/errorMessage.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+const errorMessage = (text, toCase) => {
+  const errors = [];
+  const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
- Function name should start from verb
- Move `availableCases` outside the function and the constant should be in the upper case